server_tests/test/clientTests.js

```diff
@@ -787,89 +789,151 @@ describe('RUST API TEST SUITE', function () {
     if (!isOSC2) {
       return this.skip()
     }
+    var expectedImageCount
+    var expectedVideoCount
+    var totalEntryCount
 
     before(function () {
       // delete exisiting files on SD card, if any
+      return Utility.deleteAllImages() //should have a new utility deleteAll() or something
```

Comment:
Oh I didn't think of that, I will add it. There wasn't one for listImages either. Should I add that for OSC level 1 as well?